DecipherNow/gm-fabric-dashboard
# Add API tests for the discovery-service service

## Proposed Changes
Adds API tests to the api-tests/ directory, beginning with tests for the discovery-service service. These tests require two JKS files to function correctly. Functionality will be added in another PR to pull these JKS files from an external resource at build time.

## How to Test
Once you have the expected JKS files in place, run `./gradlew apiTests` from the api-tests/ directory to run the tests.
